rs/bitcoin/ckbtc/minter/tests/tests.rs

```diff
@@ -1364,32 +1360,39 @@ fn test_min_retrieval_amount_mainnet() {
 
 #[test]
 fn test_min_retrieval_amount_testnet() {
-    let ckbtc = CkBtcSetup::new_with(Network::Testnet);
+    let min_amount = 12_345;
+    let ckbtc = CkBtcSetup::new_with(Network::Testnet, min_amount);
 
     ckbtc.refresh_fee_percentiles();
     let retrieve_btc_min_amount = ckbtc.get_minter_info().retrieve_btc_min_amount;
-    assert_eq!(retrieve_btc_min_amount, 10_000);
+    assert_eq!(retrieve_btc_min_amount, min_amount);
 
     // The numbers used in this test have been re-computed using a python script using integers.
     ckbtc.set_fee_percentiles(&vec![0; 100]);
     ckbtc.refresh_fee_percentiles();
     let retrieve_btc_min_amount = ckbtc.get_minter_info().retrieve_btc_min_amount;
-    assert_eq!(retrieve_btc_min_amount, 10_000);
+    assert_eq!(retrieve_btc_min_amount, min_amount);
 
     ckbtc.set_fee_percentiles(&vec![116_000; 100]);
     ckbtc.refresh_fee_percentiles();
     let retrieve_btc_min_amount = ckbtc.get_minter_info().retrieve_btc_min_amount;
-    assert_eq!(retrieve_btc_min_amount, 60_000);
+    assert_eq!(retrieve_btc_min_amount, 50_000 + min_amount);
```

Comment:
As fees are deducted from the withdrawal amount, so it seems correct to me that min withdrawal amount should include the fee, otherwise withdrawal will fail anyway.

Reply:
I guess this allows to potentially have lower minimum withdrawal amounts in case the transaction fees are currently low, which is contrast to ckETH, where the minimum withdrawal amount is currently a (high) constant. The ckBTC approach does totally make sense, I was just wondering why we have 2 different approaches.